pkg/network/go/goid/internal/generate_goid_lut.go

```diff
@@ -0,0 +1,161 @@
+// +build ignore
```

Comment:
Instead of doing `// +build ignore` it may be easier to move this out of the `pkg/network` folder into a new tool

Reply:
Could you explain how that would work/what benefits it would give?

Also, for context, the `generate_goid_lut.go` code is specific to the `pkg/network/go/goid` package (in that it's only used to generate the lookup tables for that package), and it's not a generic tool that could be used in other places.

Reply:
> Could you explain how that would work/what benefits it would give?

The main benefit is that it separates tool dependencies from Agent dependencies (so that we don't have issues with dependency interactions and we don't list any tool-only dependencies as part of our licensing list). If it's too hard/not desirable for some reason we can always do this later!